pom.xml

```diff
@@ -2365,21 +2338,45 @@
     </repository>
 
     <repository>
-      <!-- Contains snapshot and release (including third-party-dependencies)              -->
+      <id>osgeo-releases</id>
+      <name>OSGeo Nexus Release Repository</name>
+      <url>https://repo.osgeo.org/repository/release/</url>
+      <!-- contains release (including third-party-dependences)                            -->
       <!-- Restlet maven Repository (http://maven.restlet.org)                             -->
```

Comment:
This repo does not cache restlet.org so this line could be removed.

Reply:
I ended up with:
```
      <!-- Contains releases for OSGeo projects: GeoTools, GeoWebCache, GeoServer, ...     -->
      <!-- Includs third-party-dependences, and caches of:                                 -->
      <!-- ucar (https://artifacts.unidata.ucar.edu/content/repositories/unidata-releases) -->
      <!-- geosolutions (http://maven.geo-solutions.it/)                                   -->
```

Reply:
From https://github.com/geoserver/geoserver/pull/4165 , we ended up having to mirror restlet, so leaving this unchanged, as it is now accurate, and the repo does cache restlet.